packages/types/src/project.ts

```diff
@@ -2,22 +2,45 @@
 // SPDX-License-Identifier: GPL-3.0
 
 import {ApiWrapper} from './interfaces';
-import {StellarEvent, StellarEventFilter} from './stellar';
+import {
+  StellarBlock,
+  StellarBlockFilter,
+  StellarEffect,
+  StellarEffectFilter,
+  SorobanEvent,
+  SorobanEventFilter,
+  StellarOperation,
+  StellarOperationFilter,
+  StellarTransaction,
+  StellarTransactionFilter,
+} from './stellar';
 
 export enum StellarDatasourceKind {
   Runtime = 'stellar/Runtime',
 }
 
 export enum StellarHandlerKind {
-  Event = 'stellar/EventHandler',
+  Block = 'stellar/BlockHandler',
+  Transaction = 'stellar/TransactionHandler',
+  Operation = 'stellar/OperationHandler',
+  Effects = 'stellar/EffectHandler',
+  Event = 'soroban/EventHandler',
```

Comment:
I think this is a good idea  to keep this `soroban/`. Should there be a distinction between a stellar and soroban transaction handler?

Reply:
it won't be necessary. soroban transaction is just a stellar transaction with an operation that makes contract calls.